openmetadata-service/src/main/java/org/openmetadata/service/monitoring/NoopEventMonitor.java

```diff
@@ -0,0 +1,26 @@
+package org.openmetadata.service.monitoring;
+
+import org.openmetadata.schema.monitoring.EventMonitorProvider;
+import org.openmetadata.schema.type.ChangeEvent;
+
+public class NoopEventMonitor extends EventMonitor {
```

Comment:
Do we need `noop` or can we skip adding it a event handler if the configuration is not present?